engine/src/main/java/com/datatorrent/stram/cli/ApexCli.java

```diff
@@ -641,6 +644,10 @@ void printUsage(String cmd)
         null,
         null,
         "Exit the CLI"));
+    globalCommands.put("quit", new CommandSpec(new ExitCommand(),
```

Comment:
If there is no difference between "exit" and "quit", and both share the same CommandSpec instance, can quit be aliased to exit?